cspell.json

```diff
@@ -917,6 +927,7 @@
         "dkhttp",
         "dlsserver",
         "dmesg",
+        "doesnt",
```

Comment:
I know this probably got added for a reason, but I feel like we probably don't want to excuse instances of this without the apostrophe...

Reply:
I agree with this as well.

Reply:
I will have to agree, too. 🤷‍♂